pifpaf/drivers/__init__.py

```diff
@@ -201,7 +201,11 @@ def _exec(self, command, stdout=False, ignore_failure=False,
         if stdout or wait_for_line:
             lines = []
             while True:
-                line = c.stdout.readline()
+                try:
+                    line = c.stdout.readline()
+                except KeyboardInterrupt:
```

Comment:
Are you sure this is wide enough?
It seems to me you could have a `KeyboardInterrupt` anywhere in this function, or any other exception type.

Maybe it should be put in `finally` around the entire code to be sure it's killed?